### ISSUE TITLE
Hi, how about running Hls video sources.

### ISSUE THREAD

**Issue by Shohrux** [moay/afterglow#113]
Hi developers, i have issue with running .m3u8(HLS) sources, do your player support this type of the video or not yet, if not yet are you planning to add supporting this format of the video?

**Comment by moay** [comment]
No. For many reasons, this is currently not on the list. But maybe we'll get to this sometime. I'll reinvestigate to see if it could be added.
